src/deploy/standalone/logrotate_noobaa.conf

```diff
@@ -1,46 +1,41 @@
 /var/log/noobaa.log
 {
-    daily
     size 100M
     start 1
     missingok
     rotate 100
     compress
     create 644 root root
     sharedscripts
-    postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+    postrotate 
+        killall -HUP rsyslogd || true
     endscript
 }
 
-/var/log/client_noobaa.log
+/var/log/noobaa_events.log
 {
-    daily
-    size 100M
+    size 10M
```

Comment:
why do we need to have a different setting for noobaa_events?
others use size 100M - isn't it a good fit for this as well?

Reply:
Number of Noobaa event logs created is very less compare to normal log files because the event log will have only one entry for each events. To avoid large file with very old events entries I decided to keep a small size for the event log file rotation.

Reply:
but why did you remove the daily option then? that would have handled it

Reply:
I am ok with daily option. Like @aspandey suggested, having a size based log rotation make more sense.

Reply:
@naveenpaul1 My point was not that size will make sense more than "daily" . 
I wanted to say is that if you mention both "daily" and after that "size", then only "size" will be applicable and logrotate will not consider "daily" option.

You can check it on your laptop. Just have this conf for some file and then run logrotate command. It will say something like - "daily" is discarded/skipped as size is mentiond.
 
If you really want to include both then you have to use "min-size", "max-size" with frequency option like "daily", "monthly" or "hourly"

I have sent the patch for my use case -
https://github.com/noobaa/noobaa-core/pull/8006

Reply:
Testing with following conf file - 
/var/log/bucket_logs.log
{
        hourly
        size 10k
        start 1
        missingok
        rotate 100
        create 666 noob root
        dateext
        dateformat %Y-%m-%d-%H-%M-%S
        sharedscripts
        olddir /var/log/noobaa_logs
        postrotate
                kill -HUP $(pidof rsyslogd) >/dev/null 2>&1 || true
        endscript
}

noob@noobaa-core-0 /]$     /usr/sbin/logrotate -v /etc/logrotate.d/noobaa/logrotate_noobaa.conf >/dev/stdout 2>&1
reading config file /etc/logrotate.d/noobaa/logrotate_noobaa.conf
**warning: 'size' overrides previously specified 'daily'
warning: 'size' overrides previously specified 'daily'**
olddir is now /var/log/noobaa_logs

Reply:
so why not use max-size?